chromadb/api/fastapi.py

```diff
@@ -62,27 +62,35 @@ def __init__(self, system: System):
             default_api_path=system.settings.chroma_server_api_default_path,
         )
 
-        self._session = requests.Session()
+        self._session = httpx.Client(timeout=None)
```

Comment:
checking (seems to be yes) default behavior was previously no timeout? We should probably add config here - let me cut a ticket.

Reply:
yep, requests doesn't have a default timeout but httpx does

Reply:
the default timeout was causing issues in CI